pkg/network/usm/istio.go

```diff
@@ -22,6 +22,11 @@ import (
 	manager "github.com/DataDog/ebpf-manager"
 )
 
+const (
+	istioSslReadRetprobe  = "istio_uretprobe__SSL_read"
+	istioSslWriteRetprobe = "istio_uretprobe__SSL_write"
+)
```

Comment:
Shouldn't we have the `_ex` methods as well as best effort for future "compatibility"?

Reply:
I had a closer look and it's not that Envoy doesn't use the `_ex` methods, it's that BoringSSL doesn't have them [at all](https://commondatastorage.googleapis.com/chromium-boringssl-docs/ssl.h.html). Since we don't know if BoringSSL will ever implement such an API and what it would look like if they do decide to do it, it doesn't seem worth it to try to add it now?

Reply:
As long as it is documented (I saw you updated the RFC), we can proceed without it